src/main/java/mate/academy/internetshop/model/Bucket.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.internetshop.model;
+
+import java.util.List;
+
+public class Bucket {
+
+    private Long id;
+    private List<Item> items;
+    private Long orderId;
```

Comment:
why orderId? we need to know, whose bucket it is, so I suggest to use userId instead

Reply:
In the database come in handy "id", it will be a one-to-one relationship, and I will need to get an order as an administrator by  "id".

Reply:
@MateAcademy yeah, but user is going to make an order, not bucket, so it looks more logic

Reply:
I fixed it